### PR TITLE
Changed to display full hostname

### PR DESCRIPTION
**Pull request by caseylargent**
Addresses #172

### CODE REVIEW

File: HCCGo/app/js/navbar.js [hunk 1]
```diff
@@ -4,6 +4,5 @@ navBar.controller('NavCtrl', ['$scope', '$log', 'navService', 'connectionService
    function($scope,$log,navService,connectionService, updaterService) {
    // This controller intended purely to manage navigation bar
    // No code beyond navigational controls should be used here
-   $scope.servicer = navService; 
-   
+   $scope.servicer = navService;
```
Comment (djw8605) [inline HCCGo/app/js/navbar.js]: Odd, what's the diff here?  It looks exactly the same.

Comment (caseylargent) [inline HCCGo/app/js/navbar.js]: Changes are in the first commit, second commit was me getting rid of a console.log that I had

Comment (djw8605) [inline HCCGo/app/js/navbar.js]: Sure, that's fine.  If there is ever a bug in that line, you will be in the git-blame! :)

Comment (caseylargent) [inline HCCGo/app/js/navbar.js]: If there's a bug, it's in connection service, so I will play the git-blame game! :+1:

**Comment by djw8605** [comment]
There's some error in the test.  Does it work on your computer?  `npm test`

**Comment by djw8605** [comment]
As far as I can tell, the errors are because it's unable to load electron from HTTP.

**Pull request merged**